File: mobile-sdk/ios/payment-intent/objectivec/capture-card.m
#import <dojo_ios_sdk_drop_in_ui-Swift.h>

@property DojoSDKDropInUI *dojoUI;

self.dojoUI = [[DojoSDKDropInUI alloc] init];
[self.dojoUI startPaymentFlowWithPaymentIntentId:@"<paymentIntentId>"
                                      controller:self
                                  customerSecret:nil
                                  applePayConfig:[[DojoUIApplePayConfig alloc]
                                                     initWithMerchantIdentifier:
                                                         @"<Merchant_Identifier>"]
                                   themeSettings:nil
                                      completion:^(NSInteger result) {
                                        NSLog(@"%ld", (long)result);
}];

File: mobile-sdk/ios/payment-intent/swift/capture-card.swift
import dojo_ios_sdk_drop_in_ui

let dojoUI = DojoSDKDropInUI()
dojoUI.startPaymentFlow(paymentIntentId: "pi_sandbox_RBMHTJ4fIkmSppDILZVCGw", controller: self) {
  result in
  print(result)
}